modules/_labs/badge/react/README.md

```diff
@@ -0,0 +1,51 @@
+# Canvas Kit React Badge
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/_labs/README.md">
+  <img src="https://img.shields.io/badge/LABS-alpha-orange" alt="LABS: Alpha" />
+</a>  This component is work in progress and currently in pre-release.
+
+provides a quantity-based summary with dynamic values
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-labs-react-badge
+```
+
+## Usage
```

Comment:
Q: Should I be more descriptive about usage here? More "why" to go with the "how"?

Reply:
Usually this would go within the props below. You only have the tables, but you're missing the normal description + default that we have for each prop